lib/events/s3sessions/s3handler.go

```diff
@@ -97,6 +97,9 @@ type Config struct {
 	Insecure bool
 	// DisableServerSideEncryption is an optional switch to opt out of SSE in case the provider does not support it
 	DisableServerSideEncryption bool
+
+	// UseVirtualStyleAddressing use a virtual-hosted–style URI.
```

Comment:
Nit: I would add more info here about path vs virtual style, maybe a link to relevant AWS docs, and also mention that Teleport defaults to path style for better interoperability with 3rd party S3-compatible services out of the box.